### PR TITLE
Disable index

### PR DESCRIPTION
**Pull request by BirmacherAkos**
- Update the config parser logic to use `stepconf`

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -47,166 +46,34 @@ const (
...
+
+	UseDeprecatedExport               string `env:"use_deprecated_export,opt[yes,no]"`
+	ForceTeamID                       string `env:"force_team_id"`
+	ForceProvisioningProfileSpecifier string `env:"force_provisioning_profile_specifier"`
+	ForceProvisioningProfile          string `env:"force_provisioning_profile"`
+	ForceCodeSignIdentity             string `env:"force_code_sign_identity"`
+	CustomExportOptionsPlistContent   string `env:"custom_export_options_plist_content"`
+
+	OutputTool        string `env:"output_tool,opt[xcpretty,xcodebuild]"`
+	Workdir           string `env:"workdir,dir"`
```
Comment (godrei) [inline main.go]: workdir is not required input

File: main.go [hunk 2]
```diff
@@ -248,16 +115,21 @@ func logWithTimestamp(coloringFunc ColoringFunc, format string, v ...interface{}
...
-	configs.print()
-
-	if err := configs.validate(); err != nil {
+	var cfg configs
+	if err := stepconf.Parse(&cfg); err != nil {
 		fail("Issue with input: %s", err)
 	}
 
-	log.SetEnableDebugLog(configs.VerboseLog == "yes")
+	if cfg.CustomExportOptionsPlistContent != "" {
```
Comment (godrei) [inline main.go]: pls also restore the `auto-detect` export method warning: https://github.com/bitrise-steplib/steps-xcode-archive/pull/151/files#diff-7ddfb3e035b42cd70649cc33393fe32cL114

File: main.go (hunk 1 above)
Comment (BirmacherAkos) [reply main.go]: Fixed in:
https://github.com/bitrise-steplib/steps-xcode-archive/pull/151/commits/1978f1e243a3c3ece84cb3b785a70bb85b505e37

File: CHANGELOG.md [hunk 3]
```diff
@@ -234,7 +234,7 @@
 ### 1.7.3 (2016 May 20)
 
 * [9b50305] v1.7.3
-* [3f641bd] Merge branch 'master' of github.com:bitrise-io/steps-xcode-archive
+* [3f641bd] Merge branch 'master' of github.com:bitrise-steplib/steps-xcode-archive
```
Comment (godrei) [inline CHANGELOG.md]: instead pls remove the changelog as well, check the `Updated release flow` section of the repo revision guide

File: step.sh [hunk 4]
```diff
@@ -4,7 +4,7 @@ THIS_SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 
 tmp_gopath_dir="$(mktemp -d)"
 
-go_package_name="github.com/bitrise-io/steps-xcode-archive"
```
Comment (godrei) [inline step.sh]: pls remove this file, since we use the Go toolkit to perform the Go script

File: CHANGELOG.md (hunk 3 above)
Comment (BirmacherAkos) [reply CHANGELOG.md]: Fixed in:
https://github.com/bitrise-steplib/steps-xcode-archive/pull/151/commits/78a85e677b3e813fe5420c2aaa334e1030510ebd

File: step.sh (hunk 4 above)
Comment (BirmacherAkos) [reply step.sh]: Fixed in:
https://github.com/bitrise-steplib/steps-xcode-archive/pull/151/commits/78a85e677b3e813fe5420c2aaa334e1030510ebd

**Pull request merged**